.github/workflows/publish-nightly.yml

```diff
@@ -124,3 +124,41 @@ jobs:
           remote_port: ${{ secrets.NIGHTLIES_RSYNC_PORT }}
           remote_user: ${{ secrets.NIGHTLIES_RSYNC_USER }}
           remote_key: ${{ secrets.NIGHTLIES_RSYNC_KEY }}
+
+      - name: Build Documentation
+        run: |-
+          sbt -Dpekko.genjavadoc.enabled=true docs/paradox unidoc
+
+      # as found in https://github.com/apache/flink/blob/5a4e0ea31aef51dfc50b9c401c91c7b869a154c4/.github/workflows/docs.yml
+      - name: Upload documentation
+        uses: burnett01/rsync-deployments@5.2
```

Comment:
This may not work because Apache maintains a list of github actions that are allowed to run and I can't remember if its global or per project, @pjfanning might recall because he set up the INFRA tickets.

Reply:
just raise a JIRA issue in INFRA project. https://issues.apache.org/jira/secure/Dashboard.jspa

Reply:
I just changed it to use our own script, seems similar enough...